config/am-scripts/scripts-library/ch-library-functions.js

```diff
@@ -202,6 +202,50 @@ function _getUserInfoById (userId, accessToken) {
   }
 }
 
+// convert jurisdiction name to code
+function _getJurisdictionCode (data) {
+
+  if (!data || !data.jurisdiction) {
+    return null;
+  }
+
+  if (data.jurisdiction === 'england-wales' || data.jurisdiction === 'wales' || data.jurisdiction === 'england') {
+    return 'EW';
+  } else if (data.jurisdiction === 'scotland') {
+    return 'SC';
+  } else if (data.jurisdiction === 'northern-ireland') {
+    return 'NI';
+  } else {
+    return data.jurisdiction;
+  }
+}
+
+function _convertDateToString (date) {
+  var result = [];
+  result.push(date.getFullYear());
+  result.push(padding(date.getMonth() + 1));
+  result.push(padding(date.getDate()));
+  result.push(padding(date.getHours()));
+  result.push(padding(date.getMinutes()));
+  result.push(padding(date.getSeconds()));
+  result.push('Z');
+  return result.join('');
+}
+
+function _convertStringToDate (dateStr) {
```

Comment:
It might be helpful to just to add a one line comment to specify the correct format a date needs to be passed, so that the reader doesn't need to manually process it each time to find out?

// Format = YYYY-MM-DD